### PR TITLE
Add bash completion for common dev-lxc subcommands on containers in cwd

### PR DESCRIPTION
**Pull request by sean-horn**
Update with full command and include cookbook_file
so that something actually happens with kitchen converge

### CODE REVIEW

File: files/default/bash-completion-dev-lxc [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    #replace spaces and place in extglob format
+    pipeopts="@(`echo $opts | sed 's/ /|/g'`)"
+
+    #
+    #  Complete the arguments to some of the basic commands.
+    #
+    case "${prev}" in
+        $pipeopts)
+            local names=`dev-lxc list | grep Final | awk  '{ print $NF }'`
```
Comment (jeremiahsnapp) [inline files/default/bash-completion-dev-lxc]: Can you use the full name of the subcommand here please?

`dev-lxc list-images`

**Comment by jeremiahsnapp** [comment]
Thanks @sean-horn this looks great. I think it's missing a cookbook_file resource or something right?

**Comment by sean-horn** [comment]
I think I addressed the two comments in the previous commit with 5add846

**Comment by jeremiahsnapp** [comment]
Thanks a lot @sean-horn !

**Pull request merged**